### PR TITLE
solution

### PR DESCRIPTION
**Pull request by serentinos**
- [DEMO LINK](https://serentinos.github.io/layout_stop-watch/)

### CODE REVIEW

File: src/styles/blocks/watch.scss [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  height: $clock-max-size;
+
+  border: 10px solid black;
+  border-radius: 50%;
+  box-shadow: 0 21px 40px 0 rgba(0,0,0,0.61);
+
+  &__seconds {
+    @include time-arrow (
+      $seconds-heigh, 4px, $clock-max-size, red);
+    animation: 30s infinite steps(60, end) rotating;
```
Comment (niaYaro) [inline src/styles/blocks/watch.scss:14]: Great animation!!! 🔥

**Review by niaYaro — APPROVED** [review]
Great job! 
But as for me, watch::after should positioned a little bit to the right (as I see it is not strictly in the middle of the watch). 
But maybe, my eyes lie to me)))

Anyway, it looks very good, especial with this animation! Well done! 👍

**Comment by serentinos** [comment]
> Great job! But as for me, watch::after should positioned a little bit to the right (as I see it is not strictly in the middle of the watch). But maybe, my eyes lie to me)))
> 
> Anyway, it looks very good, especial with this animation! Well done! +1

@niaYaro Thanks! Honestly, I don't why why this ::after won't center properly. I've tried many times to fix this. If you have any suggestions about that, let me know!

**Review by Anna-Tkachenko — APPROVED** [review]
Well Done! ⭐️

**Pull request closed without merging**